### PR TITLE
LRDOCS-7380 liferay util fetch

### PR DESCRIPTION
**Pull request by mwilliams2014**
@jbalsas here is the current draft. I've added a question I had as well. Please let me know if anything needs added or removed. Thanks!

### CODE REVIEW

File: en/developer/reference/articles/02-front-end/05-liferay-js-apis/06-handling-ajax-requests.markdown [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  return response.json();
+}).then(function(response) {
+  message.innerHTML = response.message;
+}).catch(function() {
+  failureCallback();
+});
+```
+
+| **NOTE:** global access through `Liferay.Util` is only meant for use in legacy 
+| JSP code.
```
Comment (mwilliams2014) [inline en/developer/reference/articles/02-front-end/05-liferay-js-apis/06-handling-ajax-requests.markdown]: What do you mean by "legacy JSP code?"

Comment (jbalsas) [reply en/developer/reference/articles/02-front-end/05-liferay-js-apis/06-handling-ajax-requests.markdown]: Scratch `legacy`. Just meant that this code is not ES6, not transpiled, so you can't do the `import from` thingy

File: en/developer/reference/articles/02-front-end/05-liferay-js-apis/06-handling-ajax-requests.markdown [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- A thin wrapper on ES6 [`fetch`](https://fetch.spec.whatwg.org/) that shares 
+  the same API
+- Sets `credentials:include` to each request
+- Sets `x-csrf-token` header to each request
+- Requires no dependencies
+
+Below is an example configuration in ES6:
+
+```JavaScript
+import {fetch} from 'frontend-js-web';
```
Comment (jbalsas) [inline en/developer/reference/articles/02-front-end/05-liferay-js-apis/06-handling-ajax-requests.markdown]: This means we might need to start publishing our artifacts... @wincent, what do you think?

**Comment by mwilliams2014** [comment]
@jbalsas sent https://github.com/sez11a/liferay-docs/pull/4635

**Pull request closed without merging**